### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by cerebrl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,9 +52,9 @@
     "@typescript-eslint/parser": "^4.7.0",
     "@typescript-eslint/typescript-estree": "^4.7.0",
     "cookie-parser": "^1.4.4",
+    "copy-webpack-plugin": "^9.0.0",
     "copyfiles": "^2.1.1",
     "cors": "^2.8.5",
-    "cpy-cli": "^3.1.1",
```
Comment (cerebrl) [inline package.json:57]: Removed this utility and replaced it with the webpack plugin above. This one had a dependency that had a security alert.

File: tests/e2e/app/authn-webauthn/autoscript.js [hunk 2]
```diff
@@ -37,18 +37,25 @@
     // Do nothing
   }
 
-  console.log('Initiate first step with `undefined`');
+  // Needed for testing WebAuthn on Safari due to user event needed
+  console.log('Click the login button!');
+  const loginBtn = document.querySelector('.login-btn');
   rxjs
-    .from(forgerock.FRAuth.next())
+    .fromEvent(loginBtn, 'click')
```
Comment (cerebrl) [inline tests/e2e/app/authn-webauthn/autoscript.js:44]: This adds user events to accommodate the Safari needs with WebAuthn.

File: webpack.config.js [hunk 3]
```diff
@@ -19,11 +20,24 @@ module.exports = (env) => {
   const plugins = [
     new webpack.WatchIgnorePlugin({ paths: [/bundles|docs|lib|lib\-esm|samples/] }),
     new webpack.BannerPlugin({ banner }),
+    new CopyPlugin({
```
Comment (cerebrl) [inline webpack.config.js:23]: New copy mechanism for the build

File: webpack.config.js [hunk 4]
```diff
@@ -19,11 +20,24 @@ module.exports = (env) => {
...
+          info: { minimized: true },
+        },
+      ],
+    }),
     {
       apply: (compiler) => {
+        // Copy built files
         compiler.hooks.afterEmit.tap('AfterEmitPlugin', (compilation) => {
           const cmds = [
-            'cpy ./tests/e2e/env.config.ts ./tests/e2e/server --rename=env.config.copy.mjs',
```
Comment (cerebrl) [inline webpack.config.js:26]: Old copy mechanism.

**Review by spetrov — APPROVED** [review]
Looks good to me! 👍🏻

**Pull request merged**